apps/glusterfs/placer_arbiter.go

```diff
@@ -103,6 +103,12 @@ func (bp *ArbiterBrickPlacer) PlaceAll(
 		if err != nil {
 			return r, err
 		}
+		if bs.IsSparse() {
```

Comment:
The 'Sparse' naming might be confusing me. Why would a SparseBrick/Device be an error here?

Reply:
Because newSets() is supposed to return a fully populated set (i.e. one without nil entries).
These checks might be more natural at the end of newSets().